code/obj/flock/structure/sentinel.dm

```diff
@@ -64,22 +64,23 @@
 				var/mob/loopmob = null
 				var/list/hit = list()
 				var/mob/mobtohit = null
-				for(loopmob in mobs)
-					if(IN_RANGE(loopmob, src, 5) && !isflock(loopmob) && src.flock?.isEnemy(loopmob) && isturf(loopmob.loc))
+				//for(loopmob in mobs) IN_RANGE() -- this cannot be more efficient - Amylizzle
```

Comment:
```suggestion
```
Comment not needed

Reply:
confirmed. source of all byond knowledge pali says it used to be better but now isn't.